### PR TITLE
docs: translate the rest of the pages to english

### PR DESCRIPTION
**Pull request by shadowusr**

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx:1]: todo (does not need to be fixed in this pr. Just spamming with this link so it won't be forgotten): add typescript example: https://nda.ya.ru/t/aYcpLGxO77RVKU

Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx:1]: todo (does not need to be fixed in this pr. Just spamming with this link so it won't be forgotten): add typescript example: https://nda.ya.ru/t/aYcpLGxO77RVKU

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    await browser.url("https://github.com/");
+
+    const sidebar = await browser.$(".dashboard-sidebar");
+    const newRepoButton = await sidebar.getByText("New");
+
+    await newRepoButton.click();
+});
+```
+
+## Useful Links {#useful_links}
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx:59]: Typo: useful**L** links

This needs to be fixed everywhere (i found 7 instances)

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+it("example", async ({ browser }) => {
+    await browser.url("https://github.com/");
+
+    const newRepoButton = await browser.getByText("New");
+
+    await newRepoButton.click();
+});
+```
+
+In the context of found elements, this feature will also be available:
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx:46]: I would rephrase this as "This feature will also be available in the context of found elements"

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        setupBrowser(browser);
+    },
+
+    // other testplane settings...
+};
+```
+
+## Usage
+
+After connecting, you will be able to use the search by selectors from `testing-library`, as described in the [official documentation](https://testing-library.com/docs/queries/about/). For example, searching for an element by its text:
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx:34]: "After connecting". "connecting" should be translated as "configuration", "setup" or "installation"

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-check-accessibility.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import Admonition from "@theme/Admonition";
+
+# How to Test Page Accessibility
+
+## Overview {#overview}
+
+<Admonition type="warning">
+This recipe only works when using _Chrome DevTools Protocol (CDP)_.
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-check-accessibility.mdx:8]: I would add "when using CDP or chrome" in all instances

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-get-report.mdx [hunk 5]
```diff
@@ -0,0 +1,115 @@
+# How to Get a Test Run Report
+
+## testplane Report {#testplane_report}
+
+After completing all test runs, testplane writes the result to the console in the form of a string like this:
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-get-report.mdx:5]: Our team member would say "Testplane as product name should be written with upper case T, and it could be written with lower case t if it is used as cli command".

I suggest to quickly fix this everywhere replacing " testplane " with " Testplane ". This should fix most of these.

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-get-rid-of-diffs-in-screenshots.mdx
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-get-rid-of-diffs-in-screenshots.mdx:1]: Why do we need this file? Seems empty(?)

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-hide-scrollbars.mdx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+<Admonition type="warning">
+    Update the Chrome browser to version 72.1 or higher for this functionality to work in your
+    tests. Earlier versions of Chrome do not support the _Emulation.setScrollbarsHidden_ command,
+    which is used to disable the scrollbars.
+</Admonition>
+
+## Keywords {#keywords}
+
+-   screenshotDelay
+-   hermione-hide-scrollbars
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-hide-scrollbars.mdx:60]: Just curious: why do we need keywords? For CEO stuff? When i saw these, i instantly wanted to click it, but its just text :(

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-minify-screenshots.mdx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+To reduce the space screenshots occupy on disk, add the [testplane-image-minifier][testplane-image-minifier] plugin to your project. This plugin supports 8 levels of lossless image compression.
+
+How to add and configure the plugin is detailed in the [plugin documentation][testplane-image-minifier].
+
+## Keywords {#keywords}
+
+-   testplane-image-minifier
+
+[testplane-image-minifier]: ../../plugins/testplane-image-minifier
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-minify-screenshots.mdx:36]: Did you recently renamed the repo?
https://github.com/gemini-testing/testplane-image-minifier
I suggest to remove everything about "testplane-image-minifier" as it is deprecated and no longer needed since `testplane` started using sharp

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-mitigate-flaky-tests.mdx [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# How to Overcome Unstable Tests
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-mitigate-flaky-tests.mdx:1]: Why do we need these empty pages?

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-remove-useless-screenshots.mdx [hunk 9]
```diff
@@ -0,0 +1,5 @@
+# How to Delete Extra Screenshots
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-remove-useless-screenshots.mdx:1]: It would be filled later (thursday)?

File: i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+    });
+});
+```
+
+This test will immediately pass, and only after 30 seconds (assuming there are still tests in the queue to run) will an `Unhandled rejection` error appear.
+
+Therefore, to find the problematic test, you will need to look at the execution log and find the last completed tests before the error information. The error will look something like this:
+
+```bash
+[13:48:57 GMT+3] Unhandled Rejection in hermione:worker:10830:
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx:50]: `hermione:worker` -> `testplane:worker`

File: i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+-   Properly supported taking screenshots on devices with fractional `pixelRatio` (e.g., `Google Pixel`). Previously, screenshots on such devices were incorrect;
+-   Moving the cursor away using the `resetCursor: true` option no longer triggers an error if the top-left corner of the `body` element has negative coordinates.
+
+## How to Migrate? {#how_to_move}
+
+### 1. Update hermione to 7+ and Plugins to the Latest Versions {#update_hermione_and_plugins}
+
+Specifically:
+
+-   [hermione][hermione]
```
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx:88]: It would be nice to replace all of expired "hermione" references as well, but it is hard, because some of them should be "hermione". Old plugins, for example.

File: i18n/en/docusaurus-plugin-content-docs/current/commands/mock/abort.mdx
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/commands/mock/abort.mdx:1]: In browser and element commands, you left "We'd like to give credit to the original WebdriverIO docs article..." reference. Don't you want to do this as well for mock and expect commands?

File: docs/plugins/testplane-passive-browsers.mdx
Comment (KuznetsovRoman) [inline docs/plugins/testplane-passive-browsers.mdx:1]: Why is this plugin missing?
Does it exist on "ru" localization?

**Review by KuznetsovRoman — APPROVED** [review]
Other than that, /ok

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx:1]: Thanks! I have opened a PR to the main repo's examples folder with your files: https://github.com/gemini-testing/testplane/pull/990/files

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx (hunk 1 above)
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx:59]: Hmm, I am not sure what you're implying here :/
Looks correct to me.

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-get-rid-of-diffs-in-screenshots.mdx
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-get-rid-of-diffs-in-screenshots.mdx:1]: Got rid of empty files in both languages.

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-hide-scrollbars.mdx (hunk 6 above)
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-hide-scrollbars.mdx:60]: They were there in the original docs, so I kept them in the translation. It's not clear to me either (I guess originally their purpose was to give you an idea what to look for), we may get rid of them in the future.

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-mitigate-flaky-tests.mdx (hunk 8 above)
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-mitigate-flaky-tests.mdx:1]: got rid of them.

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-remove-useless-screenshots.mdx (hunk 9 above)
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-remove-useless-screenshots.mdx:1]: Removed empty pages.

File: i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx (hunk 10 above)
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx:50]: No, in fact this is accurate, because describes migration process from hermione v6 to v7

File: i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx (hunk 11 above)
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx:88]: I did replace all mentions of hermione with testplane. Hermione is here because it's a migration guide for the old version.

File: docs/plugins/testplane-passive-browsers.mdx
Comment (shadowusr) [reply docs/plugins/testplane-passive-browsers.mdx:1]: removed it as it's deprecated.

File: i18n/en/docusaurus-plugin-content-docs/current/commands/mock/abort.mdx
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/commands/mock/abort.mdx:1]: You are right. Added references in mock and expect directories.

File: i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-minify-screenshots.mdx (hunk 7 above)
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-minify-screenshots.mdx:36]: No, I didn't rename anything. There are no mentions that this plugin is deprecated...

I suggest to discuss this in the next iteration. Added this to my TODO list.
